vulcan/data/_message.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from typing import AsyncIterator, List, Union
+
+from related import ChildField, IntegerField, SequenceField, StringField, immutable
+
+from .._api_helper import FilterType
+from .._endpoints import DATA_MESSAGE
+from ..model import Attachment, DateTime, Serializable
+
+
+@immutable
+class Address(Serializable):
+    """An address.
```

Comment:
Try to be a little more descriptive, because the docs are here for a reason, to document a feature. A user might not know that this is for example not the teacher's home address, but rather a "descriptor" used in the system containing the user's Login ID, his names (including his role) and a hash of the data.